lib/diplomat/query.ex

```diff
@@ -47,7 +47,7 @@ defmodule Diplomat.Query do
 
     RunQueryRequest.new(
       query_type: {:gql_query, q |> Query.proto()},
-      partition_id: PartitionId.new(namespace_id: namespace, proejct_id: project)
+      partition_id: %PartitionId{namespace_id: namespace, project_id: project}
```

Comment:
What was the reason for this particular change? `PartitionId.new/1` should be provided by exprotobuf, and is consistent with the way we create other protobuf structs (for example `RunQueryRequest.new/1` a couple lines above).

Reply:
Sorry, just realized I should have tagged @leozilla on the above comment.

Reply:
I changed it because I got an error during runtime I think. Maybe the problem was just the misspelled `proejct_id` and it wouldnt have been necessary to change from `PartitionId.new` to `%PartitioId{..}`. I am not sure anymore.